brokerapi/brokers/gcp_service_broker.go

```diff
@@ -42,22 +41,20 @@ import (
 	"github.com/GoogleCloudPlatform/gcp-service-broker/db_service"
 )
 
+// GCPServiceBroker is a brokerapi.ServiceBroker that can be used to generate an OSB compatible service broker.
 type GCPServiceBroker struct {
 	Catalog          map[string]models.Service
 	ServiceBrokerMap map[string]models.ServiceBrokerHelper
 
 	Logger lager.Logger
 }
 
-type GCPAsyncServiceBroker struct {
-	GCPServiceBroker
-}
-
-// returns a new service broker and nil if no errors occur else nil and the error
-func New(cfg *config.BrokerConfig, Logger lager.Logger) (*GCPAsyncServiceBroker, error) {
+// New creates a GCPServiceBroker.
+// Exactly one of GCPServiceBroker or error will be nil when returned.
+func New(cfg *config.BrokerConfig, logger lager.Logger) (*GCPServiceBroker, error) {
```

Comment:
A lot of exported functions' signatures have been changed due to the `GCPAsyncServiceBroker` rename to `GCPServiceBroker`. Is this a concern for backwards compatibility? I.E would anything outside this repo break because someone used `GCPAsyncServiceBroker` instead of the `GCPServiceBroker`, or is this struct internal?